### PR TITLE
Add points implementation in DG

### PR DESCRIPTION
**Pull request by kinjalagarwal1810**
Add UML sequence diagram for addpoints command

### CODE REVIEW

File: docs/diagrams/AddPointsSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+create CommandResult
+AddPointsCommand -> CommandResult
+activate CommandResult
+
+CommandResult --> AddPointsCommand
+deactivate CommandResult
+
+AddPointsCommand --> LogicManager : r
+deactivate DeleteCommand
```
Comment (billyhoce) [inline docs/diagrams/AddPointsSequenceDiagram.puml:82]: Don't think this "DeleteCommand" should be here. The diagram ends up having a DeleteCommand object when there shouldnt be one! I think should be `deactivate AddPointsCommand`

File: docs/diagrams/AddPointsSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+create AddPointsCommandParser
+AddressBookParser -> AddPointsCommandParser
+activate AddPointsCommandParser
+
+AddPointsCommandParser --> AddressBookParser
+deactivate AddPointsCommandParser
+
+AddressBookParser -> DeleteCommandParser : parse(\n"n/Alice \np/50")
+activate AddPointsCommandParser
```
Comment (billyhoce) [inline docs/diagrams/AddPointsSequenceDiagram.puml:33]: I think `DeleteCommandParser` should be `AddPointsCommandParser` here

File: docs/diagrams/AddPointsSequenceDiagram.puml (hunk 1 above)
Comment (kab-dot) [inline docs/diagrams/AddPointsSequenceDiagram.puml:82]: I think this line is not needed, because the arrow points to DeleteCommandParser

**Review by billyhoce — COMMENTED** [review]
Overall LGTM besides the two pointers

**Review by kab-dot — COMMENTED** [review]
Other than the one comment, looks good

File: docs/diagrams/AddPointsSequenceDiagram.puml (hunk 1 above)
Comment (kinjalagarwal1810) [reply docs/diagrams/AddPointsSequenceDiagram.puml]: Thanks for pointing out. Edited!

File: docs/diagrams/AddPointsSequenceDiagram.puml (hunk 2 above)
Comment (kinjalagarwal1810) [reply docs/diagrams/AddPointsSequenceDiagram.puml:33]: Thanks for pointing out. Edited!

File: docs/diagrams/AddPointsSequenceDiagram.puml (hunk 1 above)
Comment (kinjalagarwal1810) [reply docs/diagrams/AddPointsSequenceDiagram.puml]: Thanks for pointing out. Edited!

**Pull request merged**